appshell/cef_host_window.cpp

```diff
@@ -168,7 +168,20 @@ BOOL cef_host_window::HandleSize(BOOL bMinimize)
     }
     SetProp(L"WasMinimized", (HANDLE)bMinimize);
 #endif
+    NotifyWindowMovedOrResized();
+    return FALSE;
+}
 
+void cef_host_window::NotifyWindowMovedOrResized()
+{
+    if (GetBrowser() && GetBrowser()->GetHost()) {
+        GetBrowser()->GetHost()->NotifyMoveOrResizeStarted();
```

Comment:
calling this was something that needed to be added whenever the app is moved or resized